### DISCUSSION TITLE
Project sign up

### DISCUSSION THREAD

**Discussion by Ossifragus** [STAT5125-UConn/2022Spring#12] [Announcements]
In the [project](https://github.com/STAT5125-UConn/2022Spring/tree/main/project) folder, the [signup link](https://classroom.github.com/a/MWhxJSvT) means to let you choose/propose a topic for the project. After joint the team, everyone should do this by modifying the same file [README.md](https://github.com/STAT5125-UConn/final-project-topic-sign-up-ty/blob/main/README.md). 

If you choose to introduce a Julia package, the presentation can be similar to a lecture or a workshop kind of presentation, but other appropriate presentations styles may also be allowed.

**Comment by jguil2022** [comment]
How long will we have to present our project? Or, will you be posting that information later?

**Reply by Ossifragus** [reply]
I will determine this after having a better idea on the topics everyone chooses.
